scripts/parser.js

```diff
@@ -0,0 +1,49 @@
+const jobsModel = require('../app/models/jobsModel.js');
+const MBO = require('mindbody-sdk');
+const crypto = require('crypto');
+ 
+const mbo = new MBO({
+    ApiKey: process.env.API_KEY, // from portal
+    SiteId: parseInt(process.env.SITE_ID) //thisis the sandbox account
+});
+
+/**
+ * Iterates through provided class information from Mindbody API
+ * and updates or inserts jobs to the database
+ */
+async function processResponse(err,data){
+    if (err) {
+        console.error("Failed to retrieve information about classes", err);
+    } else {
+        const classes = data.Classes;
+        await classes.map(async function(class_json) {
+            const scheduledJobs = await jobsModel.getById(class_json.Id);
+            const jobNotPresent = scheduledJobs.length === 0;
+            const job = {
+                "class_id" : class_json.Id,
+                "scheduled_time" : class_json.StartDateTime,
+            };
+            if (jobNotPresent){
+                const dateString = (new Date()).valueOf().toString();
+                const randomString = Math.random().toString();
+                const baseString = dateString + randomString;
+                job.job_hash = crypto.createHash('sha1').update(baseString).digest('hex');
+                await jobsModel.insert(job);
+            }
+            else{
+                await jobsModel.update(job);
+            }
+        });
+    }
+}
+
+/**
+ * Gets scheduled class information from Mindbody API
+ */
+async function getJobs15Mins(){
```

Comment:
I guess I made a mistake right here, the getJobs15Mins should really be called getJobIn15Mins, it should get the classes that is going to happen 15 minutes from now, and add them as a job. This will make the call back function trickier. Feel free to just work on this function for the next pull request if you want to. Right now, you can just have this pr handle the model functions.